packages/nextjs/app/brankit/page.tsx

```diff
@@ -0,0 +1,314 @@
+"use client";
```

Comment:
This file is already on develop branch. I dont expect to see this change. Let's check again of the PR is synced with develop branch

Reply:
Oops sorry, that was a mistake in the folder name, I will correct it now and upload important comments so @Buuh2511 can review it better.